chapter-A.23-struct/15-struct-type-alias.go

```diff
@@ -0,0 +1,21 @@
+package main
```

Comment:
also it would be good if the name of this file changed into `15-struct-alias-2.go`. the term *struct alias* and *struct type alias* here have the same meaning, but I'm afraid for beginner users those two different terms will cause some confusion.

another alternative suggestion, replace both example filename of 14 and 15 instead into these:
- `14-struct-type-alias-1.go`
- `15-struct-type-alias-2.go`

Reply:
Please review again, thanks.